custom/panel_templates/Default/core/users_sessions.tpl

```diff
@@ -0,0 +1,105 @@
+
+    <!-- Wrapper -->
+    <div id="wrapper">
+
+        <!-- Sidebar -->
+        {include file='sidebar.tpl'}
+
+        <!-- Content Wrapper -->
+        <div id="content-wrapper" class="d-flex flex-column">
+
+            <!-- Main content -->
+            <div id="content">
+
+                <!-- Topbar -->
+                {include file='navbar.tpl'}
+
+                <!-- Begin Page Content -->
+                <div class="container-fluid">
+
+                    <!-- Page heading -->
+                    <div class="d-sm-flex align-items-center justify-content-between mb-4">
+                        <h1 class="h3 mb-0 text-gray-800">{$USERS}</h1>
+                        <ol class="breadcrumb float-sm-right">
+                            <li class="breadcrumb-item"><a href="{$PANEL_INDEX}">{$DASHBOARD}</a></li>
+                            <li class="breadcrumb-item active">{$USER_MANAGEMENT}</li>
+                            <li class="breadcrumb-item active">{$USERS}</li>
+                        </ol>
+                    </div>
+
+                    <!-- Update Notification -->
+                    {include file='includes/update.tpl'}
+
+                    <div class="card shadow mb-4">
+                        <div class="card-body">
+                            <div class="row">
+                                <div class="col-md-9">
+                                    <h5 style="margin-top: 7px; margin-bottom: 7px;">{$VIEWING_USER_SESSIONS}</h5>
+                                </div>
+                                <div class="col-md-3">
+                                    <span class="float-md-right">
+                                        <a href="{$BACK_LINK}" class="btn btn-warning">{$BACK}</a>
+                                    </span>
+                                </div>
+                            </div>
+                            <hr />
+
+                            <!-- Success and Error Alerts -->
+                            {include file='includes/alerts.tpl'}
+
+                            <div class="table-responsive">
+                                <table class="table table-borderless table-striped">
+                                    <thead>
+                                        <tr>
+                                            <th>{$SESSION}</th>
+                                            <th>{$DEVICE}</th>
+                                            <th>{$ACTIVE}</th>
+                                            <th>{$LOGIN_METHOD}</th>
+                                            <th></th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        {foreach $SESSIONS as $session}
+                                            <tr>
+                                                <td>
+                                                    {$session.hash}
```

Comment:
A loosely held opinion here, showing the session id/hash is not required and serves no real purpose for any site admin who would be on this page. 

If we didn’t display it, the information would be small enough to possibly fit on the main user overview page

Reply:
```suggestion
                                                    <kbd€{$session.hash}</kbd>
```